tests/e2e/progressive.go

```diff
@@ -138,16 +138,9 @@ func VerifyMonoVertexRolloutScaledDownForProgressive(
 	}).Should(BeTrue())
 }
 
-// NOTE: this function assumes that the pipeline only has one source vertex.
-// This function should be modified if the E2E tests will be changed
-// to have more than one source vertex.
 func VerifyPipelineRolloutScaledDownForProgressive(
 	pipelineRolloutName string,
-	expectedPromotedName string,
-	sourceVertexName string,
-	expectedCurrent int64,
-	expectedOriginalScaleMinMaxAsJSONString string,
-	expectedScaleTo int64,
+	expectedPromotedPipelineName string,
```

Comment:
We no longer treat the Source Vertex differently from any other vertex, so it doesn't make sense to treat Source in a special way. that's why I removed those Source-related checks.

A separate issue is written and prioritized to update the e2e testing for this PR.